client/dive-common/use/useModeManager.ts

```diff
@@ -230,7 +233,11 @@ export default function useModeManager({
     _selectKey(key);
   }
 
-  function handleSelectTrack(trackId: TrackId | null, edit = false) {
+  function handleSelectTrack(
+    trackId: TrackId | null,
```

Comment:
Thing that should have been a long while ago (TrackId is fine but it is technically deprecated in the type definition since we have the base abstract Annotation class)
```suggestion
    trackId: AnnotationId | null,
```